### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by ksurmel**

### CODE REVIEW

File: docs/src/content/docs/basics/terminology.mdx [hunk 1]
```diff
@@ -12,7 +12,11 @@ A service that retrieves data from the user's *virtual agent*, anonymizes it, an
 
 ## Agent SDK
 
-Platform-independent core logic integrated by an *app* that enables them to be compatible with the *Mee data network.*  
+A software development kit that (i) provides the core logic of an *identity agent* and (ii) enables a *local* *app* to become interoperable with the *Mee data network*. The *agent SDK* manages *personal information* that may be (i) created from the user's data entry into their *identity agent* (ii) created by an app (or co-created by an *app* during interactions with the user) or (iii) received over the *Mee data network* from another *identity* *agent*.
```
Comment (ksurmel) [inline docs/src/content/docs/basics/terminology.mdx:15]: Noticed the duplication of the Agent SDK sections and replaced with the one which I liked most

File: docs/src/content/docs/basics/terminology.mdx [hunk 2]
```diff
@@ -24,20 +28,12 @@ An assertion made about a *subject* in a *context*.
 
 ## Connection
 
-A data exchange relationship between (a) an *identity agent* and (b) an *app* or *agent*. A *connection*'s state is stored by one or more *contexts*. 
+A data exchange relationship between (a) an *identity agent* and (b) an *app*. A *connection*'s state is stored by one or more *contexts*.
```
Comment (ksurmel) [inline docs/src/content/docs/basics/terminology.mdx:31]: Looks like the term "App" includes the term "Identity agent" if I get it right, suggest removing this duplication

File: docs/src/content/docs/basics/terminology.mdx [hunk 3]
```diff
@@ -24,20 +28,12 @@ An assertion made about a *subject* in a *context*.
...
 
 ## Connector
 
 A software component that implements one of potentially several aspects of a *connection*. A *connector* implements an interface (abstraction) exposed by the core of an *identity agent*. It is associated with a single *context*. It may receive data from an *app*, may store receieved data in a *context*, may read data from a *context* and may transmit data to an *app*.
 
-## Agent SDK
-
-A software development kit that (i) provides the core logic of an *identity agen*t and (ii) enables a *local* *app* to become interoperable with the *Mee data network*. The *agent SDK* manages *personal information* that may be (i) created from the user's data entry into their *identity agent* (ii) created by an app (or co-created by an *app* during interactions with the user) or (iii) received over the *Mee data network* from another *identity* *agent*.
-
-## Agent Service
```
Comment (ksurmel) [inline docs/src/content/docs/basics/terminology.mdx:37]: Just moved this section upper alphabetically

File: docs/src/content/docs/basics/terminology.mdx [hunk 4]
```diff
@@ -46,7 +42,7 @@ A container of data that holds the state for one aspect of a *connection*. The *
 
 An identity agent is an agent that represents the user on the internet. It manages the storage of personal information about the user. 
 
-Depending on the implementation variant (see *virtual identity agent* and *local identity agent*) this information may be stored on-device, and/or within an *app*, and/or by a third-party storage provider service. 
+Depending on the implementation variant (see *virtual identity agent* and *local identity agent*) this information may be stored within an *app*, and/or by a third-party storage provider service.
```
Comment (ksurmel) [inline docs/src/content/docs/basics/terminology.mdx:45]: Looks like the "App" term includes the information of being used on device, suggest removing this duplication

**Comment by ksurmel** [comment]
I think I don't get the concept of what "Mee-compatible" means from the Terminology, primarily because we say that those apps and identity agents "may or may not be on the Mee data network". Do I get it right that the identity agent is always a part of the "Mee data network"? What does the app need to do to become "Mee-compatible"? Is the term "Mee-compatible" related to the "Agent Service" term somehow?

File: docs/src/content/docs/basics/terminology.mdx (hunk 1 above)
Comment (ptrevithick) [reply docs/src/content/docs/basics/terminology.mdx:15]: I agree with this change

File: docs/src/content/docs/basics/terminology.mdx (hunk 2 above)
Comment (ptrevithick) [reply docs/src/content/docs/basics/terminology.mdx:31]: Ah, good point! Thanks. I agree with this change.

File: docs/src/content/docs/basics/terminology.mdx (hunk 3 above)
Comment (ptrevithick) [reply docs/src/content/docs/basics/terminology.mdx:37]: I agree with this change

File: docs/src/content/docs/basics/terminology.mdx (hunk 4 above)
Comment (ptrevithick) [reply docs/src/content/docs/basics/terminology.mdx:45]: I agree with this change.

**Comment by ptrevithick** [comment]
> I think I don't get the concept of what "Mee-compatible" means from the Terminology, primarily because we say that those apps and identity agents "may or may not be on the Mee data network". Do I get it right that the identity agent is always a part of the "Mee data network"? 
>> YES 

What does the app need to do to become "Mee-compatible"? 
>> It just means that it can exchange data with a Mee identity agent using some protocol(s) (which may or not be the protocols of the Mee Data Network

Is the term "Mee-compatible" related to the "Agent Service" term somehow?
>> Compatibility implies that the bits and bytes can flow in a connection (between an agent and an app) using whatever protocol (e.g. it could the Google Account API/protocol (which is NOT part of the Mee Data Network (which uses specific protocols and is associated with the Mee Data Network License))

**Pull request merged**